srcJava/com/rti/perftest/harness/PerftestPrinter.java

```diff
@@ -0,0 +1,486 @@
+                    if (_showCPU) {
+                        System.out.printf(",%1$8.2f", outputCpu);
+                    }
+                    System.out.printf("\n");
+                    System.out.flush();
+                    break;
+                case JSON:
+                    if (_printIntervals) {
+                        System.out.printf("\n\t\t\t],\n");
+                    }
+                    System.out.printf("\t\t\t\"summary\":{\n" +
+                                  "\t\t\t\t\"packets\": %1$1d,\n" +
+                                  "\t\t\t\t\"packets/sAve\": %2$1.0f,\n" +
+                                  "\t\t\t\t\"mbpsAve\": %3$1.1f,\n" +
+                                  "\t\t\t\t\"lost\": %4$1d,\n" +
+                                  "\t\t\t\t\"lostPercent\": %5$1.2f",
+                                  intervalPacketsReceived,
+                                  intervalPacketsReceived * 1000000.0
+                                         / intervalTime,
+                                  intervalBytesReceived * 1000000.0
+                                         / intervalTime * 8.0 / 1000.0 / 1000.0,
+                                  intervalMissingPackets,
+                                  missingPacketsPercent);
+                    if (_showCPU) {
+                        System.out.printf(",\n\t\t\t\t\"cpu\": %1$1.2f", outputCpu);
+                    }
+                    System.out.printf("\n\t\t\t}\n\t\t}");
+                    break;
+                case LEGACY:
+                System.out.printf("Length: %1$5d  Packets: %2$8d  " +
+                                  "Packets/s(ave): %3$7.0f  " +
+                                  "Mbps(ave): %4$7.1f  Lost: %5$5d (%6$1.2f%%)",
+                                  length,
+                                  intervalPacketsReceived,
+                                  intervalPacketsReceived * 1000000.0 /
+                                        intervalTime,
+                                  intervalBytesReceived * 1000000.0 /
+                                        intervalTime *8.0/1000.0/1000.0,
+                                  intervalMissingPackets,
+                                  missingPacketsPercent);
+                    if (_showCPU) {
+                        System.out.printf(" CPU %1$1.2f (%%)", outputCpu);
+                    }
+                    System.out.printf("\n");
+                    System.out.flush();
+                    break;
+            }
+        }
+        public void print_initial_output()
+        {
+            if (_outputFormat.equals(PerftestOutputFormat.JSON)) {
+                System.out.printf("{\"perftest\":\n\t[\n\t\t{\n");
+            }
+        }
+        public void print_final_output()
+        {
+            if (_outputFormat.equals(PerftestOutputFormat.JSON)) {
+                System.out.printf("\n\t]\n}\n");
+            }
+        }
+}
```

Comment:
Add final empty line (this is for legacy OSs)

Reply:
ok